src/OpenSage.Game/Data/SkudefReader.cs

```diff
@@ -51,7 +53,9 @@ public int CompareTo(SkudefVersion other)
 
         public static void Read(string rootDirectory, Action<string> addBigArchive)
         {
-            var skudefFiles = Directory.GetFiles(rootDirectory, "*.skudef");
+
+            var skudefFiles = Directory.GetFiles(rootDirectory).Where(filename => filename.EndsWith("skudef", StringComparison.OrdinalIgnoreCase)).ToList();
```

Comment:
If I have a file named as `RA3_english_1.12skudef` instead of `RA3_english_1.12.skudef`, I think it will be matched by `filename.EndsWith` too

Reply:
We should use the built-in overload of `Directory.GetFiles` that uses case-insensitive matching:

```suggestion
            var skudefFiles = Directory.GetFiles(rootDirectory, "*.skudef", new EnumerationOptions { MatchCasing = MatchCasing.CaseInsensitive };
```